.github/workflows/continuous-integration.yml

```
@@ -89,6 +89,11 @@ jobs:
           pip install yamllint
           python .github/scripts/ci/run_yamllint.py --config .yamllint.yaml
 
+      - name: find broken wikilinks
+        run: |
+          ARTICLES=$( git diff --diff-filter=d --name-only ${{ github.sha }}^ ${{ github.sha }} "wiki/**/*.md" "news/*.md" )
+          python .github/scripts/ci/find_broken_wikilinks.py --target $ARTICLES
```

I forgot to handle the case when there's nothing to check
```suggestion
          [[ -n "$ARTICLES" ]] && python .github/scripts/ci/find_broken_wikilinks.py --target $ARTICLES
```

---

the script itself prints a notice when that's the case

---

which means that the check will always fail if no articles are edited in a PR, for example, if I replace an image

---

oh you're right it throws if you don't provide anything to --target

---

though if you do rename an image, it wouldn't have anything to check 🤔